### PR TITLE
Completed javascript primitives. -- Joshua Preciado

### PR DESCRIPTION
**Pull request by joshuapreciado**
PRIMITIVES

### CODE REVIEW

File: javascript-primitives-and-variables/main.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+var fullName = 'Josh';
+console.log('value of fullName:', fullName);
+var isCool = Boolean(true);
```
Comment (BrettAlbright) [inline javascript-primitives-and-variables/main.js]: you don't need to use Boolean here, just `true` is fine

File: javascript-primitives-and-variables/main.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+console.log('value of isCool:', isCool);
+var totalPets = Number('2');
```
Comment (BrettAlbright) [inline javascript-primitives-and-variables/main.js]: you don't need Number here, just a number will do

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
two small updates!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**